docs/csharp/getting-started/publishing-with-visual-studio-2017.md

```diff
@@ -15,21 +15,20 @@ ms.assetid: a19545d3-24af-4a32-9778-cfb5ae938287
 
 # Publishing your Hello World application with Visual Studio 2017
 
-In [Building a C# Hello World application with .NET Core in VIsual Studio 2017 RC](with-visual-studio-2017.md), you built your Hello World console application, and in [Debugging your C# Hello World application with Visual Studio 2017 RC](debugging-with-visual-studio-2017.md), you tested it using the Visual Studio debugger. Now that you're sure that it works as expected, you an publish it so that other users can run it. Publishing creates the set of files that are needed to run your application; you can deploy them by copying them to a target machine.
+In [Building a C# Hello World application with .NET Core in Visual Studio 2017 RC](with-visual-studio-2017.md), you built your Hello World console application, and in [Debugging your C# Hello World application with Visual Studio 2017 RC](debugging-with-visual-studio-2017.md), you tested it using the Visual Studio debugger. Now that you're sure that it works as expected, you an publish it so that other users can run it. Publishing creates the set of files that are needed to run your application; you can deploy them by copying them to a target machine.
 
 Publishing .NET Core console applications from Visual Studio has not yet been implemented in the RC release. It will be implemented in the final release of Visual Studio 2017.
 
-You can publish your application using the [dotnet.exe](../../core/tools/dotnet.md) command line utility. To do this:
+You can publish your application using the [dotnet.exe](../../core/tools/dotnet.md) command-line utility. To do this:
 
 1. Open a console window. For example, in the **Ask me anything** text box in the Windows taskbar, enter `Command Prompt`, and then choose the **Command Prompt** desktop app to open the console window.
 
 1. Navigate to your library's project directory. Typically, unless you've reconfigured the file location, it is in the `Documents\Visual Studio 2017\Projects\HelloWorld\HelloWorld` directory. The directory contains your source code and a project file, `HelloWorld.csproj`.
 
-<!-- Steve, can you check this on your system? I'm sure I didn't do any renaming, but that's the directory I have on minem and I'd expect it to be HelloWorld instead of Hello. -->
```

Comment:
I'm removing this comment but is there any action item left here? @stevehoag @rpetrusha